packages/number-line/src/number-line/graph/tick-utils.js

```diff
@@ -175,20 +189,17 @@ export const fraction = (v) => {
 };
 
 export const normalizeTicks = (domain, ticks, opts) => {
```

Comment:
please keep the `normalizeTicks` as it is on develop branch because was added some improvements on the ticks

Reply:
Done.